cli-plugins/insight/query-data.hbs.md

```diff
@@ -25,135 +24,118 @@ There are four commands for querying and adding data.
+When using Supply Chain Tools - Scan 2.0, the image digest can be found by looking inside the corresponding
+image vulnerability scan custom resource.
 
-For example:
+To get a list of image vulnerability scans, run:
 
 ```console
-$ tanzu insight image get --digest sha256:407d7099d6ce7e3632b6d00682a43028d75d3b088600797a833607bd629d1ed5
-Registry:	docker.io
-Image Name:	checkr/flagr:1.1.12
-Digest:    	sha256:407d7099d6ce7e3632b6d00682a43028d75d3b088600797a833607bd629d1ed5
-Packages:
-	1. alpine-baselayout@3.1.2-r0
-	2. alpine-keys@2.1-r2
-	3. apk-tools@2.10.4-r2
-	CVEs:
-		1. CVE-2021-30139 (High)
-		2. CVE-2021-36159 (Critical)
-	4. busybox@1.30.1-r3
-	CVEs:
-		1. CVE-2021-28831 (High)
-...
+kubectl get imagevulnerabilityscan -n WORKLOAD-NAMESPACE
 ```
 
-## <a id='example2'></a>Example #2: What packages and CVEs does my source code contain?
-
-### Determining source code org, repo, and commit SHA
+For example:
 
-To query a source scan for vulnerabilities, you need a Git org and Git repository, or the commit SHA.  Find these by examining the source scan resource.
+```console
+$ kubectl get imagevulnerabilityscan -n my-apps
+NAME                                  SUCCEEDED   REASON
+tanzu-java-web-app-grype-scan-jb76m   True        Succeeded
+```
+ 
+The name of the image vulnerability scan will start with the name of the workload.
 
-Run:
+To describe the image vulnerability scan, run:
 
 ```console
-kubectl describe sourcescan WORKLOAD-NAME -n WORKLOAD-NAMESPACE
+kubectl describe imagevulnerabilityscan IMAGE-VULNERABILITY-SCAN-NAME -n WORKLOAD-NAMESPACE
 ```
 
 For example:
 
 ```console
-kubectl describe sourcescan tanzu-java-web-app -n my-apps
+kubectl describe imagevulnerabilityscan tanzu-java-web-app-grype-scan-jb76m -n my-apps
 ```
 
-In the resource look for the `Spec.Blob` field. Within, there's `Revision` and `URL`.
+In the resource, look for the `Spec.Image` field. The value points to the image that was scanned, including its digest.
 
 For example:
 
 ```yaml
 Spec:
-  Blob:
-    Revision:     master/c7e4c27ba43250a4b7c46f030355c108aa73cc39
-    URL:          http://source-controller.flux-system.svc.cluster.local./gitrepository/my-apps/tanzu-java-web-app-gitops/c7e4c27ba43250a4b7c46f030355c108aa73cc39.tar.gz
+  Image: fake.oci-registry.io/dev-cluster/supply-chain-apps/tanzu-java-web-app-my-apps@sha256:a24a8d8eb724b6816f244925cc6625a84c15f6ced6a19335121343424be693cd
 ```
 
-In the earlier example, the URL is parsed and split into the org and repo. Revision is parsed as the commit SHA.
+In this example, the image's digest is: `sha256:a24a8d8eb724b6816f244925cc6625a84c15f6ced6a19335121343424be693cd`
 
-- Org is parsed as `gitrepository`
-- Repo is parsed as `my-apps/tanzu-java-web-app-gitops/c7e4c27ba43250a4b7c46f030355c108aa73cc39.tar.gz`
-- Commit SHA is parsed as `master/c7e4c27ba43250a4b7c46f030355c108aa73cc39`
 
-Use this information to perform your search.
+### Using Supply Chain Tools - Scan Pre-2.0
```

Comment:
This is H3 but the Scan 2.0 heading is H4. Are they supposed to be the same level of heading?

Reply:
Great catch. I'll update it.